### PR TITLE
Add vac billing currency code endpoint

### PR DESCRIPTION
**Pull request by JoMas971**
@SuperConfuserUser @ariquinones @fradinni @r-chowdhury can i get a +1 on this one please ? It's pretty straight forward addition.

### CODE REVIEW

File: src/sdk/model/common/__json__/billing-currency-code-json.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { CurrencyCode } from '../billing/__json__/currency-code-type';
+
```
Comment (r-chowdhury) [inline src/sdk/model/common/__json__/billing-currency-code-json.ts]: Add doc

File: src/sdk/model/common/billing-currency-code.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* istanbul ignore next: autogenerated */
+import { BillingCurrencyCodeJson } from './__json__/billing-currency-code-json';
+import { CurrencyCode } from './billing/__json__/currency-code-type';
+
```
Comment (r-chowdhury) [inline src/sdk/model/common/billing-currency-code.ts]: doc

**Comment by r-chowdhury** [comment]
Besides those two things, it's good for me.

File: src/sdk/model/common/__json__/billing-currency-code-json.ts (hunk 1 above)
Comment (JoMas971) [reply src/sdk/model/common/__json__/billing-currency-code-json.ts]: fixed

File: src/sdk/model/common/billing-currency-code.ts (hunk 2 above)
Comment (JoMas971) [reply src/sdk/model/common/billing-currency-code.ts]: fixed

**Pull request merged**